src/Test/Utilities/Shared/Syntax/SourceUtilities.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using Microsoft.CodeAnalysis.Text;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Roslyn.Test.Utilities.Syntax
+{
+    internal sealed class RandomizedSourceText: SourceText
+    {
+        private char hash(int position)
+        {
+            // good enough "random looking" hasher via the xorshift128plus algorithm.
+            UInt64 state0 = (UInt64) 1;
```

Comment:
This does not appear to be related to the data previously processed by the test.